addons/mail/static/src/js/services/mail_notification_manager.js

```diff
@@ -169,6 +169,32 @@ MailManager.include({
             }
         }
     },
+    /**
+     * Called when receiving a join and left channel notification:
+     *
+     * @private
+     * @param {Object} channelData
+     * @param {string} channelData.channel_partner_id
+     * @param {string} channelData.channel_partner_name
+     * @param {string} channelData.name
+     */
+    _handlePartnerJoinNotification: function (channelData) {
```

Comment:
this method should be placed after `_handlePartnerChannelNotification ` (alphabetical order)